pyCGM_Single/pycgmCalc.py

```diff
@@ -42,29 +42,119 @@
 EA=SA+72*3
 
 def calcKinetics(data, Bodymass):
+    """Calculates center of mass values. 
+
+    Estimates whole body CoM in global coordinate system using PiG scaling 
+    factors for determining individual segment CoM. 
+
+    See Also
+    --------
+    pycgmKinetics.getKinetics : equivalent function; see for details.
```

Comment:
Does this hyperlink correctly in the built html docs?

Reply:
I changed it to `pyCGM_Single.pycgmKinetics.getKinetics` to get a proper hyperlink.